tests/lit.cfg

```diff
@@ -75,7 +78,7 @@ config.substitutions.append( ('%{FileCheck}', config.filecheck_path) )
 config.substitutions.append( ('%{swiftc}', config.swiftc_path) )
 config.substitutions.append( ('%{llbuild-lib-dir}', llbuild_lib_dir) )
 config.substitutions.append( ('%{srcroot}', llbuild_src_root) )
-config.substitutions.append( ('%{osx_sysroot}', config.osx_sysroot) )
+config.substitutions.append( ('%{osx_sysroot_flag}', "" if not config.osx_sysroot else "-sdk " + config.osx_sysroot) )
```

Comment:
Probably makes sense to just make this a generic "swiftc-platform-flags" substitution.